cypress/README.md

```diff
@@ -18,6 +18,17 @@ yarn install
 yarn cypress run
 ```
 
+Note that the tests expect to run against a built application, so you'll want to
+run a Docker image and either create an `app-config.cypress.yaml` or pass the
```

Comment:
Feeling that it's perhaps a bit overkill to run it in docker? Would be enough to serve the content locally too. Perhaps we can just phrase it around using your own static web server of choice in `package/app/dist` on port 7007 but use docker as a fallback?

Reply:
Is setting up a static webserver easier than running a Docker image? 🤔  The latter seems easier to me, and a bit closer to the production setup that you're wanting to test.

Reply:
Alright, can leave other methods to be discovered 😁